docs/tutorials/api/compute.rst

```diff
@@ -37,79 +36,43 @@ Optionally, login to Runhouse to sync any secrets.
 Cluster
 -------
 
+A cluster is the most basic form of compute in Runhouse, largely
+representing a group of instances or VMs connected with Ray. They
+largely fall in two categories: 1. Static clusters, which are accessed
+via IP addresses and SSH credentials. 2. On-Demand clusters, which are
+automatically spun up and down for you with your local cloud account.
+
 Runhouse provides various APIs for interacting with remote clusters,
 such as terminating an on-demand cloud cluster or running remote CLI or
 Python commands from your local dev environment.
 
 Initialize your Cluster
 ~~~~~~~~~~~~~~~~~~~~~~~
 
-There are three types of supported cluster types:
-
-1. **Bring-your-own (BYO) Cluster**: Existing clusters that you
-   already have up, and access through an IP address and SSH
-   credentials. Please refer to the :ref:`Bring-Your-Own Cluster` section for further instructions.
-
-2. **On-demand Cluster**: Associated with your cloud account (AWS, GCP,
-   Azure, LambdaLabs). There are additional features for these clusters,
-   such as cluster (auto) stop. Please refer to
-   :ref:`On-Demand Cluster` for instructions on first getting
-   cloud credentials set up.
-
-3. **SageMaker Cluster (Alpha)**: Clusters that are created and managed
```

Comment:
just checking we want to remove sagemaker from this? this tutorial is still currently acting as the main compute tutorial until we split them up